---
layout: page
excerpt: "About Me..."
---

Welcome! I am a computational biologist working in cancer research to find more precise and effective treatment options for cancer patients. In my research, I work on understanding how our cancer drugs work, how to choose the best drugs for each patient's tumor, and how to predict how patients will respond to therapies. My training in bioinformatics, cancer systems biology, and data science enables me to extract meaningful insights across tumor data types to better understand drug response. 

My domain area is druggability assessment for targeted therapies in cancer treatment, which means I learn about how targeted therapies work, how they affect tumor biology, and how resistance pathways evolve. By better understanding how drugs interact with tumor biology, we can create a better match between therapy choice and patient. For my methods area, I take what's called a pathway or systems perspective. This means I work with methods that allow us to integrate data from different levels of tumor biology (such as DNA mutation or copy number, RNA expression, and protein expression) to get a holistic view on tumor biology. The idea here is that multiple views of data can give us a) perspective on higher level patterns and b) an assessment of strength of evidence when we look at agreement across data types. 

Currently I am a postdoctoral fellow in the [Mills Lab](https://www.ohsu.edu/school-of-medicine/mills-lab/people) at the Knight Cancer Institute. I work with a collaborative team of researchers to understand how breast, ovarian, and endometrial tumors respond to the therapies we give them, and how to better drug tumors that develop resistance mechanisms to evade our drugs. Some of my projects are highlighted below; I collaborate with precision research groups in both solid and hematological cancers across the Knight. 


I'm also the Metadata Workgroup Lead for the Knight Data Governance Committee, where I get to work with an enthusiastic group of scientists to assess metadata needs and promote metadata standards for Knight research labs. This spring you'll also find me coordinating logistics for the awesome crew of [Girls Who Code PDX](https://gwcpdx.github.io/), where we run classes for 7th-12th grade girls to learn how to code.

## Research Areas
* Understanding patterns of drug sensivitiy and resistance for cancer therapies
* Prioritizing combination therapies
* Druggability assessments for patient tumor biopsies
* Drug-target interactions; off-target binding
* Pathway and network modeling methods
* Probabilistic graphical models

## Selected Projects
**Project: Using Signatures of Replication Stress to Prioritize Combination Therapies**<br/>
**Team: Mills Lab**<br/>
**Role: Project Lead**<br/>
* Responsible for project design and execution; mining for markers and signatures of replication stress in RNAseq and proteomics data across cell lines, syngeneic mouse models, and patient biopsies; includes mining public patient cohorts and internal Mills Lab datasets
* Assess concordance of existing signatures/ marker sets and integrate into candidate consensus signature
* Identify and prioritize drugs for pairing with DNA damage repair inhibitors to target replication stress deficiencies

**Project: Investigating HER2-Therapy Resistance Drivers with a Novel Live-Cell Imaging Screening Platform**<br/>
**Team Mills Lab; Collaboration with Dr. Samuel Tsang**<br/>
**Role: Collaborator and Analytics Lead**<br/>
* Responsible for data analytics planning and strategy; wrote data analytics sections for grant; [OHSU Circle of Giving Grant 2020](https://www.ohsu.edu/womens-health/circle-giving) 
* Carried out analysis for mining public databases to identify key patient HER2 mutation and co-expressed genes according to survival status  
* Project manage bioinformatics research assistant performing analytics on drug screening results from functional live-cell imaging platform  

**Project: [The Cancer Targetome](https://github.com/ablucher/The-Cancer-Targetome) ([Version 2 Beta](https://github.com/ablucher/The-Cancer-Targetome/tree/beta-V2) ), an Evidence-Based Drug Target Compendium**<br/>
**Team McWeeney Lab**<br/>
**Role: Project Lead**<br/>
* Leading expansion of Cancer Targetome resource from drug-target interaction evidence for 140 drugs to ~400 approved and investigational drugs  
* Provides target information with supporting evidence across public resources, literature, and experimental binding assay values  
* Resource supports drug screening efforts in the precision oncology efforts in the Knight: Beat AML, Head and Neck, and SMMART research teams  

**Project: A Pathway Perspective on Targeted Therapy Response in Acute Myeloid Leukemia**<br/>
**Teams: McWeeney Lab and Beat AML Consortium**<br/>
**Role: Project Lead**<br/>
* Adapt and applied a probabilistic pathway modeling framework to assess pathway impact due to somatic mutation alterations in AML patient tumor biopsies; assess pathway impact for key drugs on screening platform  
* Expanded codebase to run pathway modeling on high performance computing environment; wrote accompanying SLURM scripts  




